.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
tohle jakožto věc specifickou pro tebe bych dal jen do `.git/info/exclude`

Reply:
Ok, upravim